tests/cli/test_suite.py

```diff
@@ -676,3 +677,68 @@ def test_suite_edit_one_datasources_no_generator_with_no_additional_args(
     expected_suite_path = os.path.join(root_dir, "expectations", "my_new_suite.json")
     assert os.path.isfile(expected_suite_path)
     assert_no_logging_messages_or_tracebacks(caplog, result)
+
+
+def test_suite_list_with_zero_suites(
+    caplog, empty_data_context
+):
+    project_dir = empty_data_context.root_directory
+    context = DataContext(project_dir)
+    runner = CliRunner(mix_stderr=False)
+
+    result = runner.invoke(
+        cli,
+        "suite list -d {}".format(project_dir),
+        catch_exceptions=False,
+    )
+    assert result.exit_code == 0
+    assert ("No expectation suites available" in result.output)
+
+    assert_no_logging_messages_or_tracebacks(caplog, result)
+
+def test_suite_list_with_one_suite(
+    caplog, empty_data_context
+):
+    project_dir = empty_data_context.root_directory
+    context = DataContext(project_dir)
+    context.create_expectation_suite("a_warning")
```

Comment:
A minor preference, but I'd like to ensure our test code uses "." as the delimiter for suite names in all examples/tests to reinforce the intended hierarchical naming.